kolibri/core/exams/migrations/0002_update_exam_data_model.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2018-12-30 21:59
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('exams', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='exam',
+            name='data_model_version',
+            field=models.SmallIntegerField(default=0),
+        ),
+        migrations.AddField(
+            model_name='exam',
+            name='learners_see_fixed_order',
+            field=models.BooleanField(default=False),
+        ),
+        migrations.RenameField(model_name='exam', old_name='seed', new_name='v0_seed'),
```

Comment:
I'm beginning to re-consider this, and instead simply changing the way this value is interpreted.

A persisted seed may still come in handy later when we allow more fine-grained editing of exams, and we want the question selection to retain some consistency across edit sessions.